### ISSUE TITLE
Different background color between emacsclient -nw and emacs -nw

### ISSUE THREAD

**Issue by barischrooneyj** [hlissner/doom-emacs#3761]
First of all thanks for the amazing work!

**What did you expect to happen?**
I usually use emacs in GUI mode, but occasionally I find myself opening a file in the terminal with vim. I don't want to use two different editors so I have been trying out `emacs -nw` and `emacsclient -nw`. I expect to be able to have a consistent colour scheme with my editor and terminal, and also be able to open my editor fast.

**What actually happened?**
Doom emacs works very well in the terminal (both variations), however there is one visual aspect that is bothering me with the `emacsclient -nw`: the background color. As you can see in the image below, when I open emacs with `emacs -nw` (top right) the background color is the same as my terminal. However when I open emacs with `emacsclient -nw` (left) the background color is not the same as my terminal. The issue is highlighted further if I enable some transparency in my terminal.

<img width="1280" alt="Screenshot 2020-08-16 at 11 45 31" src="https://user-images.githubusercontent.com/6631452/90332592-08a08180-dfb6-11ea-93d5-1953e7c277b9.png">

<img width="1280" alt="Screenshot 2020-08-16 at 11 47 18" src="https://user-images.githubusercontent.com/6631452/90332618-443b4b80-dfb6-11ea-80cf-c75150c5dead.png">

Perhaps this is intended behaviour, but there must be some setting to get the background colors equal?

**Additional details:**
I run `emacs --daemon` and then in two different tmux panes I run `emacs -nw` and `emacsclient -nw` respectively.

**System information:**
<details><pre>
((emacs
  (version . "28.0.50")
  (features . "JPEG TIFF GIF PNG RSVG IMAGEMAGICK GLIB NOTIFY KQUEUE ACL GNUTLS LIBXML2 ZLIB TOOLKIT_SCROLL_BARS NS MODULES THREADS PDUMPER LCMS2")
  (build . "Aug 15, 2020")
  (buildopts "--disable-dependency-tracking --disable-silent-rules --enable-locallisppath=/usr/local/share/emacs/site-lisp --infodir=/usr/local/Cellar/emacs-plus@28/28.0.50/share/info/emacs --prefix=/usr/local/Cellar/emacs-plus@28/28.0.50 --with-xml2 --with-gnutls --without-dbus --with-imagemagick --with-modules --with-rsvg --with-ns --disable-ns-self-contained")
  (windowsys . batch)
  (daemonp . server-running))
 (doom
  (version . "2.0.9")
  (build . "HEAD -> develop 9a8fc46c 2020-08-15 15:13:50 -0400")
  (dir . "~/.doom.d/"))
 (system
  (type . darwin)
  (config . "x86_64-apple-darwin19.6.0")
  (shell . "/bin/zsh")
  (uname . "Darwin 19.6.0 Darwin Kernel Version 19.6.0: Thu Jun 18 20:49:00 PDT 2020; root:xnu-6153.141.1~1/RELEASE_X86_64 x86_64")
  (path "~/.cabal/bin" "~/.ghcup/bin" "~/.local/bin" "~/.emacs.d/bin" "~/.pyenv/shims" "/usr/local/bin" "/usr/bin" "/bin" "/usr/sbin" "/sbin" "/Applications/VMware Fusion.app/Contents/Public" "~/.cabal/bin" "~/.ghcup/bin" "~/.local/bin" "~/.emacs.d/bin" "~/.pyenv/shims" "~/.antigen/bundles/robbyrussell/oh-my-zsh/lib" "~/.antigen/bundles/robbyrussell/oh-my-zsh/plugins/docker" "~/.antigen/bundles/robbyrussell/oh-my-zsh/plugins/git" "~/.antigen/bundles/robbyrussell/oh-my-zsh/plugins/pyenv" "~/.antigen/bundles/robbyrussell/oh-my-zsh/plugins/tmux" "~/.antigen/bundles/zsh-users/zsh-autosuggestions" "~/.antigen/bundles/zsh-users/zsh-completions" "~/.antigen/bundles/zsh-users/zsh-syntax-highlighting" "~/.antigen/bundles/zsh-users/zsh-history-substring-search" "~/.antigen/bundles/romkatv/powerlevel10k" "/usr/local/Cellar/emacs-plus@28/28.0.50/libexec/emacs/28.0.50/x86_64-apple-darwin19.6.0"))
 (config
  (envfile)
  (elc-files . 0)
  (modules :completion company ivy :ui doom doom-dashboard doom-quit fill-column hl-todo indent-guides minimap modeline ophints (popup +defaults) pretty-code treemacs vc-gutter window-select workspaces :editor (evil +everywhere) file-templates fold snippets :emacs dired electric undo vc :checkers syntax :tools (eval +overlay) lookup magit :lang emacs-lisp (haskell +dante) markdown org python sh :config (default +bindings +smartparens))
  (packages (zoom))
  (unpin "n/a")
  (elpa "n/a")))
</pre></details>

**Comment by barischrooneyj** [comment]
I noticed something strange, somewhat by accident. I ran `(emacs --daemon && emacsclient -nw)` which opened Doom emacs in a tmux pane, with an opaque background, then hit `Enter` to Reload last session. This then drops me into a buffer with transparent background.

However if I jump to a second tmux pane and run `emacsclient -nw` a second buffer opens and now **both** of my emacs buffers (in separate tmux panes) become opaque.
